Backend/dev/test-tool/context-api/server.py

```
@@ -2677,8 +2680,14 @@ def _proxy(self, method):
                 self.wfile.write(resp_body)
         except urllib.error.HTTPError as e:
             resp_body = e.read()
+            if e.headers.get("Content-Encoding", "") == "gzip":
+                import gzip as _gzip
+                try:
+                    resp_body = _gzip.decompress(resp_body)
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)